pkg/lwip/Makefile

```diff
@@ -3,9 +3,33 @@ PKG_URL=git://git.savannah.nongnu.org/lwip.git
 PKG_VERSION=STABLE-2_0_3_RELEASE
 PKG_LICENSE=BSD-3-Clause
 
-.PHONY: all
+LWIP_SUBMODULES := $(patsubst makefiles/%.mk,%,$(wildcard makefiles/lwip_*.mk))
```

Comment:
Forgot about this one also in the previous PR, I would prefer another name than `SUBMODULES` because SUBMODULES means something already in the build system.